inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1978,6 +1979,15 @@ void convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function
             cnnLayer->params[ExecGraphInfoSerialization::ORIGINAL_NAMES] = originalNames;
         }
 
+        if (auto subgraph = ::ngraph::as_type_ptr<ngraph::snippets::op::Subgraph>(layer)) {
+            std::string names = "";
+            for (const auto& op : subgraph->get_body()->get_ordered_ops()) {
+                names += ", " + op->get_friendly_name();
+            }
+
+            cnnLayer->params["originalLayersNames"] += names;
+        }
+
```

Comment:
the same

Reply:
@ilya-lavrenov, @dmitry-gorokhov, I'm not sure about this. Do we still use legacy API to store executable graphs to xml file? if so, we should keep it

Reply:
done